packages/theming/src/styled/StyledBaseIcon.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import React, { Children } from 'react';
+import retrieveComponentStyles from '../utils/retrieveComponentStyles';
+import DEFAULT_THEME from '../elements/theme';
+
+const COMPONENT_ID = 'icon.base';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const StyledBaseIcon = styled(({ children, theme, ...props }) =>
+  React.cloneElement(Children.only(children), props)
```

Comment:
`props.theme` appears to be [excluded](https://github.com/styled-components/styled-components/blob/v6.1.8/packages/styled-components/src/models/StyledComponent.ts#L146) already. Was it still landing in the DOM?

Reply:
Strange, it still shows up for me 🤷🏻

Reply:
Oh, I see what's happening. `theme` is omitted in a normal `styled.div` (or any tag) call. But because of the `cloneElement`, we need to manually omit it.